app/src/main/res/layout-sw600dp-land/submitted_answer_item.xml

```diff
@@ -87,11 +87,16 @@
           android:paddingStart="12dp"
           android:paddingTop="8dp"
           android:paddingEnd="12dp"
-          android:visibility="gone"
           android:paddingBottom="8dp"
           android:text="@{submittedAnswer}"
           android:textColor="@color/oppiaPrimaryText"
-          android:textSize="16sp" />
+          android:textSize="16sp"
+          android:visibility="gone"
+          app:layout_constrainedWidth="true"
+          app:layout_constraintEnd_toEndOf="@{viewModel.isCorrectAnswer ? @id/tick_guideline : ConstraintSet.PARENT_ID}"
+          app:layout_constraintHorizontal_bias="0.0"
+          app:layout_constraintStart_toStartOf="@{!viewModel.hasConversationView}"
```

Comment:
@BenHenning This time I have used ```app:layout_constraintHorizontal_bias="0.0"``` because if I don't set this, then ```submitted_answer_textview``` appears in the middle when we constraint it to start of the parent and it already gets constrained to guideline when the answer is correct. Please do let me know if there's a better way to do this.

Reply:
Ah, in that case then I think constraining horizontal bias in this way is correct. I didn't realize that we were constraining against two rigid things here (parent start & guideline), so it will always center by default.